utoipa-axum/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Changed
 
 * Use OpenApiRouter::default for empty OpenApi (https://github.com/juhaku/utoipa/pull/1133)
+* Merge paths if they already are added to the paths map in utoipa-axum (https://github.com/juhaku/utoipa/pull/1171)
```

Comment:
Good except this has not been released yet :slightly_smiling_face:

Reply:
Lol yeah fair